homework8.py

```diff
@@ -0,0 +1,102 @@
+""" Homework8: Flow Control. Loops: writing and implementing programs
+in Python """
+
+# Быки и коровы
+# В классическом варианте игра рассчитана на двух игроков.
+# Каждый из игроков задумывает и записывает тайное 4-значное число
+# с неповторяющимися цифрами. Игрок, который начинает игру по жребию,
+# делает первую попытку отгадать число. Попытка — это 4-значное число
+# с неповторяющимися цифрами, сообщаемое противнику. Противник сообщает
+# в ответ, сколько цифр угадано без совпадения с их позициями в тайном
+# числе (то есть количество коров) и сколько угадано вплоть до позиции в
+# тайном числе (то есть количество быков). При игре против компьютера
+# игрок вводит комбинации одну за другой, пока не отгадает всю
+# последовательность.
+# Ваша задача реализовать программу, против которой можно сыграть в
+# "Быки и коровы"
+
+import random
+import string
+
+
+def play_bulls_and_cows():
+    """
+    Implement a program against which you can play "Bulls and Cows"
+    """
+
+
+digits = list(string.digits)
+random.shuffle(digits)
+RANDOM_NUMBER = ''.join(digits[:4])
+ATTEMPT = 0
+while True:
```

Comment:
it's not in function scope, add tab for all function body, that's why you should name variables in upper case

Reply:
added